crates/engraver/src/render/ir/element.rs

```diff
@@ -15,6 +19,10 @@ impl<T> Element<T> {
             Self::Polygon(polyogn) => Element::Polygon(polyogn.convert(converter)),
             Self::Symbol(symbol) => Element::Symbol(symbol.convert(converter)),
             Self::Text(text) => Element::Text(text.convert(converter)),
+            Self::Group { id, elements } => Element::Group {
+                id,
+                elements: elements.into_iter().map(|e| e.convert(converter)).collect(),
+            },
```

Comment:
… and then if there's a `Group` struct, this could live in a `convert` function on it (same as `Text`, etc.)